nl/mail/chrome/messenger/messengercompose/composeMsgs.properties

```diff
@@ -267,31 +267,31 @@ sendAnyway=Toch verzenden
 
 ## @name NS_ERROR_SMTP_AUTH_CHANGE_ENCRYPT_TO_PLAIN_NO_SSL
 # LOCALIZATION NOTE (12594): $S is server hostname
-12594=There was an error copying the message to the Drafts folder. Retry?
+12594=Er is een fout op opgetreden bij het kopiëren van het bericht naar de Concepten-map. Opnieuw proberen?
 
 ## @name NS_ERROR_SMTP_AUTH_CHANGE_ENCRYPT_TO_PLAIN_SSL
 # LOCALIZATION NOTE (12595): $S is server hostname
-12595=The SMTP server %S does not seem to support encrypted passwords. If you just set up this account, please try changing to 'Normal password' as the 'Authentication method' in the 'Account Settings | Server settings'. If it used to work and now suddenly fails, please contact your email administrator or provider.
+12595=Het lijkt erop dat de SMTP server %S geen versleutelde wachtwoorden ondersteunt. Probeer, als u dit account net heeft ingesteld, te wijzigen naar 'Normaal wachtwoord' als 'Authenticatiemethode' in de 'Account Instellingen | Server instellingen'. Neem, als het eerst wel werkte en nu opeens niet meer, contact op met uw e-mail administrator of provider.
 
 ## @name NS_ERROR_SMTP_AUTH_CHANGE_PLAIN_TO_ENCRYPT
 # LOCALIZATION NOTE (12596): $S is server hostname
-12596=The SMTP server %S does not allow plaintext passwords. Please try changing to 'Encrypted password' as the 'Authentication method' in the 'Account Settings | Server settings'.
+12596=De SMTP server %S staat geen wachtwoorden in platte tekst toe. Probeer te wijzigen naar 'Versleuteld wachtwoord' als 'Authenticatiemethode' in de 'Account Instellingen | Server instellingen'.
 
 ## @name NS_ERROR_SMTP_AUTH_FAILURE
 # LOCALIZATION NOTE (12597): $S is server hostname
-12597=Unable to authenticate to SMTP server %S. Please check the password, and verify the 'Authentication method' in 'Account Settings | Outgoing server (SMTP)'.
+12597=Niet is staat om te authenticeren naar SMTP server %S. Controleer alstublieft het wachtwoord en verifieer de 'Authenticatiemethode' in de 'Account Instellingen | Uitgaande server (SMTP)'.
```

Comment:
CORRECTION NEEDED, it should be: Niet in staat om te authenticeren naar SMTP server %S. Controleer alstublieft het wachtwoord en verifieer de 'Authenticatiemethode' in de 'Account Instellingen | Uitgaande server (SMTP)'.

- typo: ''Niet in staat'' vs ''Niet is staat''